build.proj

```diff
@@ -23,6 +23,7 @@
     <Project Include="external\dir.proj" />
     <Project Include="src\ref.builds" />
     <Project Include="src\src.builds" />
+    <Project Include="src\tests.builds" />
```

Comment:
I forgot about that. I will add a separate project, and a separate build-tests script.